src/main/java/org/mule/extension/sftp/internal/connection/SftpLogger.java

```diff
@@ -6,14 +6,13 @@
  */
 package org.mule.extension.sftp.internal.connection;
 
-import com.jcraft.jsch.Logger;
+
 import org.slf4j.LoggerFactory;
 
-public class SftpLogger implements Logger {
+public class SftpLogger {
```

Comment:
Please explain the usage of this class?

Reply:
This class is not being use I am removing the class